bridetalk_project/src/components/molecule/forms/createuserform/CreateUserForm.jsx

```diff
@@ -0,0 +1,71 @@
+import {
+  Headline1,
+  EnterText,
+  EnterEmail,
+  EnterPassword,
+  DropdownLocation,
+  PrimaryButton,
+  CancelButton,
+} from "../../../atoms";
+
+import "./createuserform.css";
+import { Link } from "react-router-dom";
+
+import { useNavigate } from "react-router-dom";
+export const CreateUserForm = () => {
+  let navigate = useNavigate();
+
+  const toFrontPage = () => {
+    let path = "/";
+    navigate(path);
+  };
+
+  const toLoginPage = () => {
+    let path = "/login";
+    navigate(path);
+  };
+
+  const goBack = () => {
+    let path = navigate(-1);
+    navigate(path);
+  };
+
+  return (
+    <div>
+      <div className="form">
+        <Headline1 headline="Sign up to Bride Talk"></Headline1>
+        <div
+          handleClick={toLoginPage}
+          className="question"
+          style={{
+            paddingBottom: "15px",
+            paddingTop: "0px",
+          }}
+        >
+          Already a user?{" "}
+          <span
+            className="login"
+            style={{
+              fontWeight: "bold",
+              textDecoration: "underline",
+            }}
+          >
+            <Link to="/login"> Log in here </Link>
+          </span>
+        </div>
+        <EnterText text="First Name "></EnterText>
+        <EnterText text="Last Name "></EnterText>
+        <EnterEmail email="Email "></EnterEmail>
+        <EnterPassword password="Password"></EnterPassword>
+        <EnterPassword password="Confirm Password"></EnterPassword>
```

Comment:
Here you can see what I mean regarding the comments I made in the EnterText, EnterEmail and EnterPassword components. Should be 1 component that accepts different props.